### DISCUSSION TITLE
v34.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3242] [Release Discussions]
<h3>🚌🚴 Quest presets</h3>
  <ul>
  <li>Show quests that are quick and easy to solve (from a distance) first by default. This makes
      contributing more efficient and diverse while also easier to do as e.g. a passenger (#2944, ...)</li>
  <li>You can now save different quest presets for different situations, e.g. one preset when you
     only want a solve some important quests while on a bus, or on a bike tour etc.<br/>This was
     a much requested feature: (#1301, #1654, #1746, #1987, #2054, #2069, #2279, #2565, #2308, #3034, #3216, ...)</li>
  <li>Inform new users once about the possibility to change quest presets</li>
  <li>Add search/filter function for the quest presets settings (#3008)</li>
  </ul>
  <h3>New Quests</h3>
  <ul>
  <li><i>Are there curbs where this way meets this road here? What kind?</i> (#398, #2999)</li>
  </ul>
  <h3>Enhancements</h3>
  <ul>
  <li>Ask whether a road has a cycleway when the current tagging is invalid (#3206, #3148)</li>
  <li>The quests that ask whether roads, ways etc. are lit are now only shown at night (#1285, #2872), by @TurnrDev</li>
  <li>The roof shape quest is no longer enabled by default - you can enable it in the settings (#3161), by @matkoniecz</li>
  <li>Clearer wording when explaining the team mode (#3170, #3172) by @matkoniecz and @smichel17</li>
  <li>Ask for more disused places whether they are no longer vacant (#3158)</li>
  <li>Add Animism as choice for when asked about the religion of a place (#3063)</li>
  <li>Add more building type choices: allotment house, boathouse and grandstand (#3078)</li>
  <li>Stone stiles are now more clearly selectable (#3175) by @matkoniecz</li>
  <li>More minor quest enhancements and clarified wordings: (#3139) by @john-h-kastner, (#3117) by @smichel17, (#3165) by @yrtimiD, (#3194) by @ZeLonewolf, (#3200) by @matkoniecz (#3092, #3149, #2760, #3116...)</li>
  </ul>
  <h3>Fixes</h3>
  <ul>
  <li>Fix layout for right-to-left languages (Persian, Arabic) (#3181, #3184)</li>
  <li>Fix steps ramp quest could seemingly in rare cases not be answered (#3115)</li>
  <li>Fix tapping compass rotates map back to North even when currently searching for GPS location (#3147, #3166), by @smichel17</li>
  <li>Fix crash when displaying opening hours whose range ended at 12 AM</li>
  <li>Fix couldn't press OK anymore after pressing cancel in the dialog that asks if you are really on-site (#3198)</li>
  <li>Fix crash when asking for the religion of a place (#3217)</li>
  <li>Clear user statistics properly when logging out (#3168)</li>
  <li>Remove location pointer pin from UI when GPS position is lost (#3213), by @smichel17</li>
  <li>In the edit history, scroll to item when it is selected on the map (#3226)</li>
  <li>Fix a few more crashes</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v34.0'>v34.0</a>.</em>

**Comment by westnordost** [comment]
I wonder if it is possible to disable expanding the issue number to the short description. It's hardly possible to read the changelog this way.

**Reply by smichel17** [reply]
I don't know of a way to turn it off for lists, but it's only done for lists, so you could prevent it by reformatting somehow, to not use a list.

**Reply by smichel17** [reply]
Oh, I guess also you create the link manually, e.g. `[#3213](https://github.com/streetcomplete/StreetComplete/pull/3213)`

**Comment by smichel17** [comment]
There is a bug in the changelog. It reads, "Fix tapping compass rotates map back to North even when currently searching for GPS location", when it should be something like "Fix tapping compass rotates with phone direction even when still searching for GPS location".

**Comment by Swyter** [comment]
Hi @westnordost. Neat release. Very small thing; I was testing the translation improvements and noticed that the Google Play descriptions aren't up to date. ¯\\\_(ツ)_/¯

**Reply by westnordost** [reply]
Oh ok, updated them!
